Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -1,96 +1,594 @@
+  desc.addUntracked<bool>("useJetTrigger",      false);
+  desc.addUntracked<double>("drLeadJetVeto",    1.2);
+  desc.addUntracked<double>("ptMinLeadJet",     15.0);
+  desc.addUntracked<int>("debugTracks",         0);
+  desc.addUntracked<bool>("printTrkHitPattern", true);
+  desc.addUntracked<double>("minTrackP",        1.0);
+  desc.addUntracked<double>("maxTrackEta",      2.6);
+  desc.addUntracked<double>("maxNearTrackP",    1.0);
+  desc.addUntracked<bool>("debugEcalSimInfo",   false);
+  desc.addUntracked<bool>("applyEcalIsolation", true);
+  desc.addUntracked<bool>("debugL1Info",        false);
+  desc.add<edm::InputTag>("L1extraTauJetSource",edm::InputTag("l1extraParticles","Tau"));
+  desc.add<edm::InputTag>("L1extraCenJetSource",edm::InputTag("l1extraParticles","Central"));
+  desc.add<edm::InputTag>("L1extraFwdJetSource",edm::InputTag("l1extraParticles","Forward"));
+
+  edm::ParameterSetDescription desc_TrackAssoc;
+  desc_TrackAssoc.add<double>("muonMaxDistanceSigmaX",0.0);
+  desc_TrackAssoc.add<double>("muonMaxDistanceSigmaY",0.0);
+  desc_TrackAssoc.add<edm::InputTag>("CSCSegmentCollectionLabel",edm::InputTag("cscSegments"));
+  desc_TrackAssoc.add<double>("dRHcal",9999.0);
+  desc_TrackAssoc.add<double>("dREcal",9999.0);
+  desc_TrackAssoc.add<edm::InputTag>("CaloTowerCollectionLabel",edm::InputTag("towerMaker"));
+  desc_TrackAssoc.add<bool>("useEcal",true);
+  desc_TrackAssoc.add<double>("dREcalPreselection",0.05);
+  desc_TrackAssoc.add<edm::InputTag>("HORecHitCollectionLabel",edm::InputTag("horeco"));
+  desc_TrackAssoc.add<double>("dRMuon",9999.0);
+  desc_TrackAssoc.add<std::string>("crossedEnergyType","SinglePointAlongTrajectory");
+  desc_TrackAssoc.add<double>("muonMaxDistanceX",5.0);
+  desc_TrackAssoc.add<double>("muonMaxDistanceY",5.0);
+  desc_TrackAssoc.add<bool>("useHO",false);
+  desc_TrackAssoc.add<bool>("accountForTrajectoryChangeCalo",false);
+  desc_TrackAssoc.add<edm::InputTag>("DTRecSegment4DCollectionLabel",edm::InputTag("dt4DSegments"));
+  desc_TrackAssoc.add<edm::InputTag>("EERecHitCollectionLabel",edm::InputTag("ecalRecHit","EcalRecHitsEE"));
+  desc_TrackAssoc.add<double>("dRHcalPreselection",0.2);
+  desc_TrackAssoc.add<bool>("useMuon",false);
+  desc_TrackAssoc.add<bool>("useCalo",true);
+  desc_TrackAssoc.add<edm::InputTag>("EBRecHitCollectionLabel",edm::InputTag("ecalRecHit","EcalRecHitsEB"));
+  desc_TrackAssoc.add<double>("dRMuonPreselection",0.2);
+  desc_TrackAssoc.add<bool>("truthMatch",false);
+  desc_TrackAssoc.add<edm::InputTag>("HBHERecHitCollectionLabel",edm::InputTag("hbhereco"));
+  desc_TrackAssoc.add<bool>("useHcal",true);
+  desc_TrackAssoc.add<bool>("usePreshower",false);
+  desc_TrackAssoc.add<double>("dRPreshowerPreselection",0.2);
+  desc_TrackAssoc.add<double>("trajectoryUncertaintyTolerance",1.0);
+  desc.add<edm::ParameterSetDescription>("TrackAssociatorParameters",desc_TrackAssoc);
+  descriptions.add("isolatedTracksCone",desc);
```

Comment:
please comment on the default parameters

Reply:
Cannot provide justification - they were used as default while evaluating the use of IsoTracks in calibration

Reply:
Ok, I think that's fine then